### ISSUE TITLE
Lab Report 5 Feedback

### ISSUE THREAD

**Issue by shad674** [ZiyuWang0113/cse15l-lab-reports#6]
Here is your feedback for Lab Report 5:

- [x] Explains how tests with different results were found (`diff`, manual search, etc)

Test 1:

- [x] Describes which, if any, implementation is correct by showing both actual outputs and indicating what the expected output is
- [ ] Describes bug in code for one of the incorrect implementation and shows the code that should be fixed.
   - did not show which code that should be fixed 

Test 2:

- [x] Describes which, if any, implementation is correct by showing both actual outputs and indicating what the expected output is
- [ ] Describes bug in code for one of the incorrect implementation and shows the code that should be fixed.
    - did not show which code that should be fixed  

Additional comments from your grader:

Feel free to comment below any questions about the rubric/feedback

--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by ZiyuWang0113** [comment]
Hi, I am not sure what should I put for those two tests about "which code should be fixed", this is my explanation:
1. In those tests, my code has the problem that "did NOT check for a certain situation, so it's a little bit unclear for me to say which part needs to fix, but I should say add a check for what condition. So, if I have to state which code should be fixed, I think I would just use "add more condition checks after line 43", which is not much useful I think?
2. I don't think I need to change or fix something IN my code but need to add more. So can the grader provide any standard form for this checkmark if the code needs something to add-in?

**Comment by shad674** [comment]
yeah similar to what you said in (1.) you should look for a test that you know will produce a wrong output, and then from there say which part of your code tells us that it fails

**Comment by shad674** [comment]
also provide screenshot of the part that you know will produce these bugs

**Comment by ZiyuWang0113** [comment]
Thank you. Report updated with screenshots and explanations.

**Comment by shad674** [comment]
good job!
